linera-explorer/src/lib.rs

```diff
@@ -36,10 +42,17 @@ enum Page {
     Blocks(Vec<Blocks>),
     Block(Box<Block>),
     Applications(Vec<Applications>),
+    Application {
+        app: Applications,
```

Comment:
```suggestion
        apps: Applications,
```

(Or `applications`?)

This is the list of all registered applications, right? Not the list of dependencies of this particular application?

Reply:
no this is just one application, the type is misleading because it is defined by the derivation of the graphql query.
I will be better if I import it `as Application`